Example/StripeTerminal.framework/Headers/SCPConnectConfiguration.h

```diff
@@ -0,0 +1,41 @@
+//
+//  SCPConnectConfiguration.h
+//  StripeTerminal
+//
+//  Created by Catriona Scott on 8/6/19.
+//  Copyright © 2019 Stripe. All rights reserved.
+//
+//  Use of this SDK is subject to the Stripe Terminal Terms:
+//  https://stripe.com/terminal/legal
+//
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ Configuration for connecting to readers.
+ */
+
+NS_SWIFT_NAME(ConnectConfiguration)
+@interface SCPConnectConfiguration : NSObject
```

Comment:
This still has the ConnectConfiguration rather than ConnectionConfiguation, that should be in this PR right?